docs/sources/articles/networking.md

```diff
@@ -499,7 +505,8 @@ want to configure `eth0` via Router Advertisements you should set:
 ![](/article-img/ipv6_basic_host_config.svg)
 
 Every new container will get an IPv6 address from the defined subnet. Further
-a default route will be added via the gateway `fe80::1` on `eth0`:
+a default route will be added on `eth0` via the address specified by the daemon
```

Comment:
I think both are taking about what gateway will be added to the `eth0` interface in the container, the one from `--default-gateway-v6` if specified, `fe80::1` otherwise. Actually, the second reference to `eth0` comes from the existing documentation. I'm not 100% though, as I don't know the IPv6 setup at all.